ido-vertical-mode.el

```diff
@@ -88,6 +88,21 @@ so we can restore it when turning `ido-vertical-mode' off")
           (const :tag "C-p/up, C-n/down are up/down in match. left or right cycle history or directory." C-n-C-p-up-down-left-right))
   :group 'ido-vertical-mode)
 
+(defface ido-vertical-first-match-face
+    '((t (:background "#e5b7c0")))
+  "Face used by Ido Vertical for highlighting first match."
+  :group 'lispy-faces)
+
+(defface ido-vertical-only-match-face
+    '((t (:foreground "white" :background "#e52b50")))
+  "Face used by Ido Vertical for highlighting only match."
+  :group 'lispy-faces)
+
+(defface ido-vertical-match-face
+    '((t (:foreground "#b00000")))
+  "Face used by Ido Vertical for highlighting only match."
+  :group 'lispy-faces)
+
```

Comment:
Adding the ability to customize the faces is definitely awesome, but I think it makes more sense for these customizations to be in a user's own .emacs.d, and in the main repository we keep the defaults plain and not change them from their defaults. I'm not sure what the face defaults are be for these, though.